docs/competition/challenges.rst

```diff
@@ -110,15 +110,15 @@ Flipped parts can be detected in one of two ways.
 Dropped Part Challenge
 ----------------------
 
-The faulty gripper challenge simulates a faulty gripper which can drop a part after the part has been picked up. The gripper can drop a part at any time during the trial. The gripper can drop a part that is in the gripper's grasp even if the gripper or robot is not moving. 
+The dropped part challenge simulates a faulty gripper which can drop a part after the part has been picked up. The gripper can drop a part at any time during the trial. The gripper can drop a part that is in the gripper's grasp even if the gripper or robot is not moving. 
 
 The goal of this challenge is to test the ability of the CCS to: 
   
   #. Recognize that the part has dropped from the gripper. 
   #. Pick a part of the same type and color.
 
-Dropped Part Detection
-======================
+Faulty Gripper Detection
```

Comment:
Should this be dropped part?

Reply:
I wasn't sure on this one. The detection is detecting a faulty gripper which causes the dropped part, so either could work?

Reply:
The rest of the sections are named after the challenge so I think it should stay dropped part detection